components/brave_perf_predictor/browser/BUILD.gn

```diff
@@ -0,0 +1,49 @@
+import("//brave/build/config.gni")
+import("//brave/components/p3a/buildflags.gni")
+
+source_set("browser") {
+  public_deps = [
+    "buildflags",
+  ]
+
+  sources = [
+    "bandwidth_savings_predictor.cc",
+    "bandwidth_savings_predictor.h",
+    "perf_predictor_page_metrics_observer.cc",
+    "perf_predictor_page_metrics_observer.h",
+    "perf_predictor_tab_helper.cc",
+    "perf_predictor_tab_helper.h",
+    "predictor.cc",
+    "predictor.h",
```

Comment:
mb `brave_perf_predictor`? we already have a bunch of different predictors in chromium